pkg/gateway/model_build_targetgroup.go

```diff
@@ -404,6 +441,57 @@ func (t *latticeServiceModelBuildTask) buildTargetGroupName(_ context.Context, b
 	}
 }
 
+func getAttachedTargetGroupPolicy(ctx context.Context, k8sClient client.Client, svcName, svcNamespace string) (*v1alpha1.TargetGroupPolicy, error) {
+	policyList := &v1alpha1.TargetGroupPolicyList{}
+	err := k8sClient.List(ctx, policyList)
+	if err != nil {
+		return nil, err
+	}
+	for _, policy := range policyList.Items {
+		targetRef := policy.Spec.TargetRef
+		if targetRef == nil {
+			continue
```

Comment:
Should we log or is this expected?

Reply:
I think we can ignore this, my ground rule here is to only care about TargetGroupPolicy that is relevant.